src/main/java/org/pesmypetcare/webservice/thirdpartyservices/adapters/firestore/FirestoreCollection.java

```diff
@@ -23,6 +25,13 @@
      */
     WriteBatch batch();
 
+    /**
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [First sentence should end with a period.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5492924)